Package.swift

```diff
@@ -2,7 +2,8 @@
 import PackageDescription
 
 let swiftSettings: [SwiftSetting] = [
-    .enableUpcomingFeature("StrictConcurrency")
+    .enableUpcomingFeature("StrictConcurrency"),
+    .unsafeFlags(["-require-explicit-sendable"]),
```

Comment:
Can't leave an unsafe flag in a versioned library package

Reply:
```suggestion
```